bacs/src/main/java/com/adyen/checkout/bacs/BacsDirectDebitConfiguration.kt

```diff
@@ -63,13 +69,23 @@ class BacsDirectDebitConfiguration private constructor(
             clientKey
         )
 
+        /**
+         * TODO docs
+         */
+        override fun setSubmitButtonEnabled(isSubmitButtonEnabled: Boolean): Builder {
+            this.isSubmitButtonEnabled = isSubmitButtonEnabled
+            @Suppress("UNCHECKED_CAST")
```

Comment:
Do we really need to suppress this? The `setAmount` function works the same way and doesn't suppress anything